internal/api/observability/secrets.go

```diff
@@ -11,3 +17,83 @@ func NewSecretReference(keyName, secretName string) *obs.SecretReference {
 		SecretName: secretName,
 	}
 }
+
+// Secrets is a map of secrets
+type Secrets map[string]*corev1.Secret
```

Comment:
We have many mentions of "Secret" in different forms already. Maybe `SecretBundle` or something like that to have a different name for it?

Reply:
Team consensus is leave as-is